src/hooks/custom-hooks/usePoiPoaStatus.ts

```diff
@@ -1,17 +1,19 @@
 import { useMemo } from 'react';
 import { useGetAccountStatus } from '@deriv-com/api-hooks';
+import { api } from '..';
 
 /** A custom hook that returns the POA, POI status and if POA is required for P2P */
 const usePoiPoaStatus = () => {
     const { data, ...rest } = useGetAccountStatus();
+    const { data: p2pSettings } = api.settings.useSettings();
 
     // create new response for poi/poa statuses
     const modifiedAccountStatus = useMemo(() => {
         if (!data) return undefined;
 
         const documentStatus = data?.authentication?.document?.status;
         const identityStatus = data?.authentication?.identity?.status;
-        const isP2PPoaRequired = data?.p2p_poa_required;
+        const isP2PPoaRequired = !!p2pSettings?.poa_required;
```

Comment:
BE had informed me they will be deleting p2p_poa_required in get_account_status so they added it to p2p_settings now under poa_required